src/resources/locales/en.json

```diff
@@ -291,26 +291,26 @@
         "remainingTime": "Loans (remaining days)",
         "renewButton": "Renew",
         "returnButton": "Return",
-        "returnedLcp": "This book can't be read because the LCP licence has been returned.",
-        "revokedLcp": "This book can't be read because the LCP licence has been revoked.",
+        "returnedLcp": "This LCP-protected publication is returned.",
+        "revokedLcp": "This publication's LCP license is revoked.",
```

Comment:
better -> "The LCP license was revoked". (to be even with expiredLCP)

Reply:
in English I feel that expressing the "state" of the publication/license is clearer than expressing the past "action" that may have resulted in the current "state" (users / readers might not necessarily taken the "action" themselves, but at least the "state" is clear and definite).